src/styles/main.css

```diff
@@ -1 +1,6 @@
-@import "./popup.css";
```

Comment:
As @mswinkels09 mentioned, there is already styling for the popup on the master branch. Please don't delete the import for popup.css